test/integration/status_test.go

```diff
@@ -99,3 +101,39 @@ func verifyClusterState(t *testing.T, contents []byte) {
 		}
 	}
 }
+
+// TestInvalidService makes sure minikube will not start a tunnel for a unavailable service who has no running pods
+func TestInvalidService(t *testing.T) {
```

Comment:
instead of creating a whole new minikube cluster for this test, we can add a subtest in functioanl test